components/brave_wallet/browser/json_rpc_service.cc

```diff
@@ -2399,6 +2399,70 @@ void JsonRpcService::OnGetSupportsInterface(
                           "");
 }
 
+void JsonRpcService::GetNftStandard(
+    const std::string& contract_address,
+    const std::string& chain_id,
+    const std::vector<std::string>& remaining_interfaces,
```

Comment:
Instead of copying this list in brave_wallet_service and pass in here, how about we use an index param that's default to 0, and increase it when calling GetNftStandard below? The stop condition will be checking the index against interfaces size, and both GetNftStandard and OnGetNftStandard will take `const std::vector<std::string>& interfaces`.